main.rs

```diff
@@ -34,3 +36,13 @@ fn get_configuration() -> Config {
 fn print_ascii_logo() {
     println!("{ASCII_LOGO}");
 }
+
+fn setup_abort_on_panic() {
+    std::panic::set_hook({
+        let default_panic = std::panic::take_hook();
+        Box::new(move |info| {
+            default_panic(info);
+            std::process::exit(1);
```

Comment:
Note: `abort()` would bail immediately without calling any destructors or exit hooks. `exit()` is more gentle.

Reply:
Ok, thank you!